src/ducks/Signals/SignalFormManager/SignalMaster.js

```diff
@@ -3,7 +3,7 @@ import { push } from 'react-router-redux'
 import { graphql } from 'react-apollo'
 import { compose } from 'recompose'
 import { connect } from 'react-redux'
-import { createTrigger, toggleTrigger, updateTrigger } from '../Redux/actions'
+import { createTrigger, updateTrigger } from '../Redux/actions'
```

Comment:
@GarageInc We use the small first letter for folders.

Reply:
will rename